Assets/Scripts/EventBus/EventBusEnum.cs

```diff
@@ -10,6 +10,8 @@ public enum EventName
         UIScoreUpdate,
         UIHealthUpdate,
         UIEndScreenUpdate,
+        UICountdownUpdate,
```

Comment:
Consider adding the values to the enums, e.g:

```
public enum EventName 
{
    StartMatch = 1,
    EndMatch = 2,

    UIScoreUpdate = 3,
    UIHealthUpdate = 4,
    UIEndScreenUpdate = 5,
    UICountdownUpdate = 6,
    UIMatchTimerUpdate = 7,

    HitEffect = 8,
    DamageEffect = 9,
    ShootEffect = 10,
    DeathEffect = 11,

    TankDead = 12,
    SpawnEnemy = 13,
}
```

The reason for this is that enums are integers, and unless you explicitly set their value, they're automatically assigned by their order. Then the problem is when you save them in another location, like a database, you will save their numeric value, but if you modify the enum order (as you did here) the value saved on a database would no longer be valid. If you explicitly set their values you can reorder them without errors.

As a side note it is also not recommended to remove enum values, or reuse their numbers, only do it when you're absolutely sure you'll have no side effects.